src/main/java/club/sk1er/hytilities/tweaker/asm/GuiScreenTransformer.java

```diff
@@ -0,0 +1,45 @@
+package club.sk1er.hytilities.tweaker.asm;
+
+import club.sk1er.hytilities.tweaker.transformer.HytilitiesTransformer;
+import org.objectweb.asm.Opcodes;
+import org.objectweb.asm.tree.*;
+
+import java.util.ListIterator;
+
+public class GuiScreenTransformer implements HytilitiesTransformer  {
+    @Override
+    public String[] getClassName() {
+        return new String[]{"net.minecraft.client.gui.GuiScreen"};
+    }
+
+    @Override
+    public void transform(ClassNode classNode, String name) {
+        for (MethodNode method : classNode.methods) {
+            String methodName = mapMethodName(classNode, method);
+            InsnList instructions = method.instructions;
+            ListIterator<AbstractInsnNode> iterator = instructions.iterator();
```

Comment:
instead of creating a new list and iterator for every single method, move these into if `if (methodName...)` block, and make any possible local variable `final`